docs/overview.rst

```diff
@@ -1,39 +1,44 @@
 Introduction
 ============
 
-Let's summarize what is included by default and some future aims for the `libdoug`:
-
-* Hub/Registry
-
-  - Get API `token` from Hub upon authenticating
-  - `Token` cached for 2 hours
-  - Also gives `ro` access for public registries
-  - Public registries are hidden behind special user `stackbrew`
-  - Fetch all tags in a repository
-
-* Updates
-
-  - Handle local/remote updates to a repository
-  - Repo diffs, Image trees, Conflict solvers ...
-
-    + Version-Release conflict solver
-    + Works kinda like RPMs
-    + Push/Pull the highest version, remove the rest
-
-* Docker API and diagnostics tools
-
-  - Got a 4 line long docker command trying to figure out what it does?
-
-    + or need a sensible format for logging? See the example of `docker-cli` sub-command above
-  - Programmatic definitions of the following Docker APIs: CLI, Docker socket v.1.16, Registry, Hub
-
-    + Autogenerated with (and manually fine-tuned): https://github.com/shaded-enmity/docker-devour
-  - The `dependencies` sub-command can be used for visualizing deeply nested hierarchies of images
-
-* What's in the works
-
-  - Toolchain for propagation of information about updates in the infrastructure
-
-    + When a new `Version-Release` tag appers in the remote, we should rebase all local children on top of the new one
-    + That means editing `Dockerfile`'s and issuing `build` requests
-    + Or editing of `k8s` pod `YAML` s
+Let's summarize what is included by default and some future aims for the `rebase-helper`:
+
+This tool helps you to rebase package to the latest version
+
+How the rebase-helper works
+---------------------------
+
+- Each action should be logged and visible by user.
+    - extract tarball with the existing sources to directory old_sources/<package_name>
+    - extract tarball with the new sources to directory new_sources/<package_name>
+    - Provide a list of patches mentioned in SPEC file
+    - New spec file is stored in results/
+- for all patches
+    - apply all patches with git command to old_sources/<package_name> directory
+    - add new_sources<package_name> to old_sources/<package_name>
+        with command ``git rebase --onto new_sources <inital_commit> <last_commit>``
+    - solve all conflicts which arise during the ``git rebase``
+- create srpm from old and new spec files & new sources & patches (Builder)
+- rebuild srpm -> RPMs (Builder)
+- Run rpmdiff tool for finding libraries and header changes.
+- Inform user what libraries and header files were changed.
```

Comment:
Please be consistent in:
- starting a sentence with uppercase or lowercase letter
- ending a sentence with full stop